app/web/src/components/AssetDetailIntrinsicInput.vue

```diff
@@ -10,32 +10,36 @@
       display.funcId === identityFuncId ? 'input-socket' : 'circle-slash'
     "
     :disabled="isLocked || display.funcId === unsetFuncId"
+    :showCautionLines="display.funcId === unsetFuncId"
```

Comment:
Didn't bundle this in with `disabled` for `VormInput` since it has users everywhere. I don't love this, but at least it is explicit and doesn't happen as a consequence of another property changing inside `VormInput`.